lib/fauxpaas/artifact_builder.rb

```diff
@@ -44,6 +45,18 @@ def add_reference(ref, path)
         .write
     end
 
+    def install_local_gems(path)
+      Bundler.with_clean_env do
+        Dir.chdir(path) do
+          Fauxpaas.system_runner.run("bundle package")
```

Comment:
It doesn't look like the cache_path is going to apply here. I think we need to supply the BUNDLE_CACHE_PATH as somewhere outside of the build. You can't supply it via parameter to `bundle package`. It has to either by in the user's .bundle/config or environment variable. I think we should use the env var.

Reply:
I should say: the reason for this is that `vendor/cache` is the default, and we definitely don't want the raw .gem files in there as part of the artifact.

Reply:
I think the key thing is that here we aren't specifying a path for the cache—this way we can have puppet set that up, making the decision on a server-by-server basis.

Reply:
We want to cache the gems on the deploy host. There's only ever going to be one of those.